src/taskgraph/util/taskcluster.py

```diff
@@ -367,3 +368,22 @@ def list_task_group_incomplete_tasks(task_group_id):
             params = {"continuationToken": resp.get("continuationToken")}
         else:
             break
+
+
+def get_recursive_dependencies(
```

Comment:
What about `get_ancestors()`?

(feel free to ignore this bikeshed)

Reply:
Yeah, that's a much better name.